tests/tvtropes_crawler_test.py

```diff
@@ -0,0 +1,21 @@
+import requests
+
+from cliche.services.tvtropes.crawler import fetch_link
+
+
+def test_fetch_link(monkeypatch, fx_session, fx_celery_app):
+
+    url = 'http://tvtropes.org/pmwiki/pmwiki.php/Main/GodJob'
+    text = '<div class="pagetitle"><div class="article_title"><h1>' \
+           '<span>God Job</span></h1></div></div>'
+
+    def mockreturn(path):
+        req = requests.Request()
```

Comment:
monkeypatch changes "request" so that the test can fetch pre-defined "request".
`mockreturn` define 'request', the test don't need any external network.

it doesn't need `response', because crawling don't need to  change response.